docs/userguide/storagedriver/device-mapper-driver.md

```diff
@@ -412,10 +445,12 @@ The `Data Space` values show that the pool is 100GB total. This example extends
 1. List the sizes of the devices.
 
 	```bash
-	$ sudo ls -lh /var/lib/docker/devicemapper/devicemapper/
+	$ sudo ls -al /var/lib/docker/devicemapper/devicemapper/
 	total 1175492
-	-rw------- 1 root root 100G Mar 30 05:22 data
-	-rw------- 1 root root 2.0G Mar 31 11:17 metadata
+	drwx------ 2 root root         4096 Mar 29 02:45 .
```

Comment:
could you change this back? the example below also uses `lh` (probably easier to read)